install.sh

```diff
@@ -370,80 +380,155 @@ if [ "$1" = "install" ]; then
+      if [ "$_distro" = "Fedora" ]; then
+        sudo dnf install $_headers_rpm $_kernel_rpm $_kernel_devel_rpm
+      elif [ "$_distro" = "Suse" ]; then
+        msg2 "Some files from 'linux-glibc-devel' will be replaced by files from the custom kernel-hearders package"
+        msg2 "To revert back to the original kernel headers do 'sudo zypper install -f linux-glibc-devel'"
+        sudo zypper install --replacefiles --allow-unsigned-rpm $_headers_rpm $_kernel_rpm $_kernel_devel_rpm
+      fi
+
+      msg2 "Install successful"
     fi
 
-  elif [ "$_distro" = "Generic" ]; then
+  elif [[ "$_distro" =~ ^(Gentoo|Generic)$ ]]; then
 
     ./scripts/config --set-str LOCALVERSION "-${_kernel_flavor}"
 
-    if make ${llvm_opt} -j ${_thread_num}; then
+    if [[ "$_sub" = rc* ]]; then
+      _kernelname=$_basekernel.${_kernel_subver}-${_sub}-$_kernel_flavor
+    else
+      _kernelname=$_basekernel.${_kernel_subver}-$_kernel_flavor
+    fi
 
-      if [[ "$_sub" = rc* ]]; then
-        _kernelname=$_basekernel.${_kernel_subver}-${_sub}-$_kernel_flavor
-      else
-        _kernelname=$_basekernel.${_kernel_subver}-$_kernel_flavor
+    msg2 "Building kernel"
+    _timed_build make ${llvm_opt} -j ${_thread_num}
+    msg2 "Build successful"
+
+    if [ "$_STRIP" = "true" ]; then
+      echo "Stripping vmlinux..."
+      strip -v $STRIP_STATIC "vmlinux"
+    fi
+
+    _headers_folder_name="linux-$_kernelname"
+
+    msg2 "Removing unneeded architectures..."
+    for arch in arch/*/; do
+      [[ $arch = */x86/ ]] && continue
+      echo "Removing $(basename "$arch")"
+      rm -r "$arch"
+    done
+
+    msg2 "Removing broken symlinks..."
+    find -L . -type l -printf 'Removing %P\n' -delete
+
+    msg2 "Removing loose objects..."
+    find . -type f -name '*.o' -printf 'Removing %P\n' -delete
+
+    msg2 "Stripping build tools..."
+    while read -rd '' file; do
+      case "$(file -bi "$file")" in
+        application/x-sharedlib\;*)      # Libraries (.so)
+          strip -v $STRIP_SHARED "$file" ;;
+        application/x-archive\;*)        # Libraries (.a)
+          strip -v $STRIP_STATIC "$file" ;;
+        application/x-executable\;*)     # Binaries
+          strip -v $STRIP_BINARIES "$file" ;;
+        application/x-pie-executable\;*) # Relocatable binaries
+          strip -v $STRIP_SHARED "$file" ;;
+      esac
+    done < <(find . -type f -perm -u+x ! -name vmlinux -print0)
+
+    echo -e "\n\n"
+
+    msg2 "The installation process will run the following commands:"
+    echo "    # copy the patched and compiled sources to /usr/src/$_headers_folder_name"
+    echo "    sudo make modules_install"
+    echo "    sudo make install"
+    echo "    sudo dracut --force --hostonly ${_dracut_options} --kver $_kernelname"
+    echo "    sudo grub-mkconfig -o /boot/grub/grub.cfg"
+
+    msg2 "Note: Uninstalling requires manual intervention, use './install.sh uninstall-help' for more information."
+    read -p "Continue ? Y/[n]: " _continue
```

Comment:
I don't know why, but sometimes the installer just refuses to continue past this point even as I enter `Y`. e.g.:
```
 -> Note: Uninstalling requires manual intervention, use './install.sh uninstall-help' for more information.
Continue ? Y/[n]: Y
 -> exit cleanup done
```
Maybe there's a better failsafe version of the prompt that can trim spaces, invisible characters, etc.

Reply:
Can you print the value of `_continue` in that `if` block, with quotes (to see white spaces), to see what's wrong ?